pkg/cmd/pulumi/templates/org.go

```diff
@@ -48,6 +48,11 @@ func (s *Source) getOrgTemplates(
 		return
 	}
 
+	creds, err := ws.GetStoredCredentials()
+	if creds.Current == "" {
```

Comment:
Is this the correct/best way to check this?

Reply:
It's not ideal, we might have an ACCESS_TOKEN set so we would be able to login fine if we tried. I think you might actually need to call into DefaultLoginManager.current and see if it errors or not.